### PR TITLE
Add linux packaging for the kvm2 driver binary

### PR DESCRIPTION
**Pull request by afbjorklund**
Just some copy/paste of the minikube packaging, should probably be cleaned up later.

For #2982

### CODE REVIEW

File: installers/linux/deb/kvm2_deb_template/DEBIAN/control [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Package: docker-machine-driver-kvm2
+Version: --VERSION--
+Section: base
+Priority: optional
+Architecture: amd64
+Depends: libvirt0 (>= 1.3.1)
+Recommends: minikube
+Maintainer: Aaron Prindle <aaprindle@gmail.com>
```
Comment (tstromberg) [inline installers/linux/deb/kvm2_deb_template/DEBIAN/control]: This should probably be someone else. If you don't want your name, you may put in my own address: `t+minikube@stromberg.org`

I wonder if Debian accepts mailing lists as an alternative to a single person?

File: installers/linux/deb/kvm2_deb_template/DEBIAN/control [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+Description: Machine driver for KVM
+ Minikube uses Docker Machine to manage the Kubernetes VM so it benefits
```
Comment (tstromberg) [inline installers/linux/deb/kvm2_deb_template/DEBIAN/control]: I prefer 'minikube' to 'Minikube'. We are mini, after all. :)

File: installers/linux/deb/kvm2_deb_template/DEBIAN/control (hunk 1 above)
Comment (afbjorklund) [reply installers/linux/deb/kvm2_deb_template/DEBIAN/control]: Okay, will change the maintainer on both packages (minikube and docker-machine-driver-kvm2)

I think that it does, the `docker-ce` package has: `Maintainer: Docker <support@docker.com>`

File: installers/linux/deb/kvm2_deb_template/DEBIAN/control (hunk 2 above)
Comment (afbjorklund) [reply installers/linux/deb/kvm2_deb_template/DEBIAN/control]: Weird to start a sentence with a lower case letter, but OK

**Pull request merged**